src/nft/components/bag/BagFooter.tsx

```diff
@@ -0,0 +1,106 @@
+import { BigNumber } from '@ethersproject/bignumber'
+import ethereumLogoUrl from 'assets/images/ethereum-logo.png'
+import Loader from 'components/Loader'
+import { Box } from 'nft/components/Box'
+import { Column, Row } from 'nft/components/Flex'
+import { bodySmall, header2, subheadSmall } from 'nft/css/common.css'
+import { BagStatus } from 'nft/types'
+import { ethNumberStandardFormatter, formatWeiToDecimal } from 'nft/utils/currency'
+import { useModalIsOpen, useToggleWalletModal } from 'state/application/hooks'
+import { ApplicationModal } from 'state/application/reducer'
+
+import * as styles from './BagFooter.css'
+
+interface BagFooterProps {
+  balance: BigNumber
+  isConnected: boolean
+  sufficientBalance: boolean
+  totalEthPrice: BigNumber
+  totalUsdPrice: number
+  bagStatus: BagStatus
+  setBagStatus: (status: BagStatus) => void
+  fetchReview: () => void
+  assetsAreInReview: boolean
+}
+
+export const BagFooter = ({
+  balance,
+  isConnected,
+  sufficientBalance,
+  totalEthPrice,
+  totalUsdPrice,
+  bagStatus,
+  setBagStatus,
+  fetchReview,
+  assetsAreInReview,
+}: BagFooterProps) => {
+  const toggleWalletModal = useToggleWalletModal()
+  const walletModalIsOpen = useModalIsOpen(ApplicationModal.WALLET)
+
+  const isPending =
+    bagStatus === BagStatus.FETCHING_ROUTE ||
+    bagStatus === BagStatus.FETCHING_FINAL_ROUTE ||
+    bagStatus === BagStatus.CONFIRMING_IN_WALLET ||
+    bagStatus === BagStatus.PROCESSING_TRANSACTION ||
```

Comment:
When >= 3 conditionals how about we go with the convention of doing an array of bagStatuses and check if it `includes` bagStatus?